### ISSUE TITLE
Add password manually

### ISSUE THREAD

**Issue by jiggleypodgery** [marius-wieschollek/passwords-webextension#148]
Forgive me if I'm being really stupid here, but how do you add a password manually through the extension?

Many passwords aren't picked up automatically (by password managers in general), but the passwords need to be recorded all the same. From what I can tell, it's only possible to do this by launching the full NC app.

I'm using Brave with the official Chrome extension.

Thanks!

**Comment by RJ-Make** [comment]
I was just going to ask this question. It appears, at least in the FF extension, you can't edit or manually add a password through the extension.

**Comment by roadriverrail** [comment]
Indeed, I'm not seeing a way to do this?  Must be common to all variants of the extension, as I'm on Chrome and using the Chrome extension.

**Comment by nowheretobefound** [comment]
It would be nice if the "Add credentials for current tab" was always visible, because the extension picks up all kinds of random things and then doesnt display the button when you actually want to add a password.

**Comment by fairking** [comment]
The entire User Interface in my opinion is a little bit frustrating.

**Comment by cptMikky** [comment]
I agree that the button should be visible all the time.

**Comment by TtuxX** [comment]
I also have this issue, why not always show the "Add credentials for current tab" ?

I have to delete collected passwords to add a custom password for current tab :(
